lib/data_model/tax_bracket.rb

```diff
@@ -0,0 +1,15 @@
+class TaxBracket
+    attr_accessor :lower_limit, :upper_limit, :rate, :is_inclusive
+
+    @lower_limit
+    @upper_limit
+    @rate
+    @is_inclusive
```

Comment:
actually in ruby if you decalre it as `attr_accessor` you dont need to add line 4-7 magically you can access those